### PR TITLE
reasonable training time

### PR DESCRIPTION
**Pull request by maximlopin**
https://github.com/mindsdb/mindsdb_native/issues/216

### CODE REVIEW

**Comment by mindsdbadmin** [comment]
****CLA Assistant Lite bot**** All contributors have signed the CLA  ✍️

File: mindsdb_native/libs/backends/lightwood.py [hunk 1]
```diff
@@ -318,6 +320,14 @@ def train(self):
                 mixer_classes = [use_mixers]
         else:
             mixer_classes = lightwood.mixers.BaseMixer.__subclasses__()
+            if self.transaction.lmd['stop_training_in_x_seconds'] is not None:
+                if self.transaction.lmd['stop_training_in_x_seconds'] > reasonable_training_time:
+                    mixer_classes = lightwood.mixers.BaseMixer.__subclasses__()
+                elif reasonable_training_time / 10 < self.transaction.lmd['stop_training_in_x_seconds'] < reasonable_training_time:
+                    mixer_classes = [lightwood.mixers.NnMixer]
+                elif reasonable_training_time / 10 > self.transaction.lmd['stop_training_in_x_seconds']:
```
Comment (George3d6) [inline mindsdb_native/libs/backends/lightwood.py]: Your debug flag comes in handy here, only do this in "debug" mode, aka when testing, but not when we release.

Comment (George3d6) [reply mindsdb_native/libs/backends/lightwood.py]: Actually, probably better to just comment this out from now, and let's get back to it after the Precona release.

File: mindsdb_native/libs/backends/lightwood.py [hunk 2]
```diff
@@ -340,7 +350,12 @@ def train(self):
 
                 kwargs['callback_on_iter'] = self.callback_on_iter
                 kwargs['eval_every_x_epochs'] = eval_every_x_epochs / len(mixer_classes)
-                kwargs['stop_training_after_seconds'] = self.transaction.lmd['stop_training_in_x_seconds']
+                
+                if self.transaction.lmd['stop_training_in_x_seconds'] is not None:
+                    if self.transaction.lmd['stop_training_in_x_seconds'] > reasonable_training_time:
+                        kwargs['stop_training_after_seconds'] = self.transaction.lmd['stop_training_in_x_seconds'] / 2
+                    elif reasonable_training_time / 10 < self.transaction.lmd['stop_training_in_x_seconds'] < reasonable_training_time:
+                        kwargs['stop_training_after_seconds'] = self.transaction.lmd['stop_training_in_x_seconds']
```
Comment (George3d6) [inline mindsdb_native/libs/backends/lightwood.py]: This logic should happen during the previous if/else chain when the mixers are selected

**Review by George3d6 — CHANGES_REQUESTED** [review]

**Comment by George3d6** [comment]
Ok, since this needs to be deployed fast I went ahead and made all the changes, @maximlopin review tomorrow and let me know of what you think

**Comment by maximlopin** [comment]
looks good

**Pull request merged**